src/lib.rs

```diff
@@ -44,4 +44,5 @@ pub(crate) use aerugo_x86 as arch;
 #[cfg(feature = "use-aerugo-x86")]
 pub use x86_hal as hal;
 
-pub use arch::log;
+#[cfg(feature = "log")]
+pub use arch::{log, logln};
```

Comment:
There should probably be a stub-empty implementation if the feature is disabled for the case we would have any `log!` in the rtos code

Reply:
done, added a stub implementation of `log` and `logln` that's compatible with stdlib's `print`/`println` (it has to be, otherwise the code may not compile because of missing macro matches)